canvas_account_admin_tools/templates/canvas_account_admin_tools/dashboard_account.html

```diff
@@ -4,68 +4,75 @@
 
 {% block stylesheets %}
 <link href="{% static 'canvas_account_admin_tools/css/dashboard.css' %}" rel="stylesheet" />
+  <style>
```

Comment:
sure. 
there was a note in the wiki "UI comment - inline style needed to get buttons of similar width and to wrap with elegance and grace. This will be resolved with a new Bootstrap v4 style"

I added that comment along with the css to the dashboard.css